snuba/query/expressions.py

```diff
@@ -142,7 +150,8 @@ class FunctionCall(Expression):
     """
 
     function_name: str
-    parameters: Sequence[Expression]
+    # This is a tuple with variable size and not a Sequence to enforce it is hashable
+    parameters: Tuple[Expression, ...]
```

Comment:
This syntax is new to me — does this mean "tuple with string repeated indefinitely", or "tuple with a first string element any `Any` repeated indefinitely"? I assume/hope the first one?

Reply:
It means: Tuple with 0 or many elements of type Expression (I tried to pass an empty tuple, ok, and a tuple with an expression and an integer did not work).
https://mypy.readthedocs.io/en/latest/kinds_of_types.html#tuple-types

Reply:
Ah, nice, I hadn't seen that before; and yeah, I should have said 0 or more. 😬  (I also copied this comment too from a block further down when I realized it was also defined above, hence the string/`Expression` mismatch, oops.)